### ISSUE TITLE
Support for Apache Spark 2.X

### ISSUE THREAD

**Issue by thesuperzapper** [SeldonIO/seldon-server#35]
Hey guys, I see that the current version of seldon-server uses Spark 1.5.2 by default, given the dramatic performance and feature improvements in Spark 2.X, are there any plans to support it?

**Comment by cliveseldon** [comment]
Yes. We would like to. Adding it instead of 1.5.2 in the kubernetes setup should be trivial change to the Pod JSONs to newer packages. We welcome your tests if you try this. Apart from that we would need to test the existing code/algos we have built into Seldon.
